src/types/User.ts

```diff
@@ -7,6 +8,18 @@ export interface User {
     isRamad: boolean;
 }
 
+export interface RamadAds {
+    id: number;
+    name: string;
+    role: RoleAdPropertiesData;
+    candidate: Condidate[];
```

Comment:
`candidates: Candidate[]`

1. plural (Could be 2+ candidates, that's why the 's' in the end 😀)
2. Typo, `Candidate` and not `Condidate`

Reply:
Thanks done